[RA2 CH1-3] fix ra2 :refs: (#3037)
cntt-n/CNTT: doc/ref_arch/kubernetes/chapters/chapter01.rst

The intention of this Reference Architecture is to develop a usable Kubernetes based platform for the Telecom operator

> community. The RA will be based on the standard Kubernetes platform where ever possible. This Reference Architecture

Reviewer: ```suggestion
community. The RA will be based on the standard Kubernetes platform wherever possible. This Reference Architecture
```

for Kubernetes will describe the high level system components and their interactions, taking the

> :ref:`common/chapter00:overview` and mapping them to real-world Kubernetes (and related)

Reviewer: If referring to the chapter and not a specific section then you don't need the ":overview"

Author: should be fixed now, replaced with doc

Terminology
~~~~~~~~~~~

> For terminology used in this document refer to the :doc:`common/glossary`.

Reviewer 2: ```suggestion
For terminology used in this document refer to the :ref:`common/glossary:glossary`.
```

> For terminology used in this document refer to the :ref:`common/glossary:glossary`.

Reviewer: ```suggestion
For terminology used in this document refer to the :ref:`common/glossary`.
```

Author: for some reason
```
:ref:`common/glossary`
```
(without `:section`) doesn't build, but
```
:doc:`common/glossary`
```
does

Exceptions
~~~~~~~~~~

Anuket specifications define certain policies and :ref:`common/chapter00:anuket general principles` and strive to
coalesce the industry towards conformant Cloud Infrastructure technologies and configurations. With the currently
available technology options, incompatibilities, performance and operator constraints (including costs), these
policies and principles may not always be achievable and, thus, require an exception process. These policies
describe how to handle :ref:`common/policies:anuket project policies for managing non-conforming technologies`.
In general, non-conformance with policies is handled through a set of exceptions (please also see

> :ref:`gov/chapters/chapter09:exception types`.

Reviewer 2: ```suggestion
:ref:`gov/chapters/chapter09:exception types`).
```
